docs/security-council-manager.md

```diff
@@ -0,0 +1,59 @@
+- **SecurityCouncilNomineeElectionGovernor.createElection** - called at T+0 where T is a multiple of 6 months after the first election date
+- **SecurityCouncilNomineeElectionGovernor.execute** - called at T+21 days
+- **SecurityCouncilMemberElectionGovernor.execute** - called at T + 42 days
+- **ArbitrumTimelock.execute** (on L2) - called at T + 42 days
+- **Outbox.executeTransaction** - called at T + 49 days
+- **L1ArbitrumTimelock.executeBatch** - called at T + 52 days
+- Retryable ticket execution on ArbOne and Nova - also called at T + 52 days
+
+
+## Public functions
+
+### Replace cohort
+
+Can only be called by the Member Election Governor. It is used to replace a whole cohort of (6) members. This is the function that will be called for the standard elections that take place every 6 months.
+
+### Remove member
+
+Can be called by the Emergency Security Council and the Removal Governor. Is used to remove a member. The Constitution allows members to be removed if 9 of 12 members wish them to be, or if the DAO votes to remove a member and 5/6 of voted tokens are in favour of removal.
+
+### Add member
+
+Can only be called by the Emergency Security Council. Can only be called if the there are less than 12 members in the Security Council because one has been removed.
+
+### Replace member
+
+Can only be called by the Emergency Security Council. This is a utility function to allow the council to call Remove and Add in the same transaction. Semantically this means that an entity has been removed from the council, and a different one has been added.
+
+### Rotate member
+
+Can only be called by the Emergency Security Council. Functionally this is the same as Replace, however semantically it infers different intent. Rotate should be called when a entity wish to replace their address, but it is the same entity that controls the newly added address.
+
+## Race conditions
+Since the Security Council Manager can be updated from a number of sources race conditions can occur in the updates. Some of these updates are long lived processes (the elections), so care must be taking to avoid these kinds of race conditions.
+
+Below we'll explain the possible sources of races, and how they're mitigated.
+
+### Standard elections
+Standard elections take place every 6 months, and last 42 days before replacing the cohort in the Manager. The election governors do checks to ensure that a contender will not become a member of both cohort, and this is then later enforced in Manager. However, whilst the elections are ongoing the membership in the Manager may be manipulated causing the result of the election to conflict (eg by adding a potential nominee to the previous cohort in the manager). This would cause the election execution to revert.
```

Comment:
```suggestion
Standard elections take place every 6 months, and last 42 days before replacing the cohort in the Manager. The election governors do checks to ensure that a contender will not become a member of both cohorts, and this is then later enforced in Manager. However, whilst the elections are ongoing the membership in the Manager may be manipulated causing the result of the election to conflict (eg by adding a potential nominee to the previous cohort in the manager). This would cause the election execution to revert.
```